### PR TITLE
Added NodeBB

### PR DESCRIPTION
**Pull request by yelinaung**
Added NodeBB link, source code link and license

### CODE REVIEW

**Comment by nodiscc** [comment]
@yelinaung can you rename the "Social Networks" section to "Social Networks and Forums"? don't forget the table of contents

**Comment by yelinaung** [comment]
@nodiscc I've made the necessary changes. Thanks!

File: README.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ See also [Static Site Generators](#static-site-generators) and  [Content Managem
   * [Isso](http://posativ.org/isso/) - a lightweight commenting server written in Python and JavaScript. It aims to be a drop-in replacement for Disqus ([Source code](https://github.com/posativ/isso)) `MIT` `Python`
   * [Jappix](https://jappix.com/) - Jappix is an open social platform, that let's you easily get or keep in touch with everyone.
   * [Movim](https://movim.eu/) - A brand new social network, full of awesome features in a modern and smart interface
+  * [NodeBB] (https://nodebb.org/) - Node.js based forum software built for the modern web ([Source code](https://github.com/NodeBB/NodeBB)) - `GPLv3`
```
Comment (nodiscc) [inline README.md]: [NodeBB] (https://nodebb.org/) wrong link syntax
Please move Node.js at the end of the line between brackets.

Comment (yelinaung) [inline README.md]: The description from the project says `Node.js based forum software built for the modern web`. How should I move Node.js at the end of the line between brackets ?

**Comment by Kickball** [comment]
@nodiscc are we alright to merge?

**Comment by nodiscc** [comment]
@Kickball  Yes this looks ok https://github.com/Kickball/awesome-selfhosted/pull/82/files

**Pull request merged**